components/CustomTag.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Mixins } from "@cheapreats/react-ui";
+import { BaseStyles } from "@cheapreats/react-ui";
+import { TagProps } from '@cheapreats/react-ui';
+
+export const CustomTag: React.FC<TagProps> = ({
```

Comment:
Question, how come you didn't extend the existing tag component with the animation feature?

Reply:
I originally create a CustomTag component because I was unable to disable the 'X' icon without directly modifying Tag.tsx. I wasn't sure if I was allowed to edit existing accepted components. I will fix this and extend Tag directly.

Reply:
You are allowed to edit existing components as long as you don't break existing implementations ;)